src/megatron/bridge/models/qwen_vl/modelling_qwen3_vl/attention.py

```
@@ -0,0 +1,260 @@
+                inference_value_memory=inference_value_memory,
+                rotary_cos=rotary_pos_cos,
+                rotary_sin=rotary_pos_sin,
+                rotary_interleaved=self.config.rotary_interleaved,
+            )
+            out = output.transpose(0, 1).contiguous()
+            context_layer = out.view(out.size(0), out.size(1), -1)
+            output, bias = self.linear_proj(context_layer)
+            return output, bias
+
+        if in_decode_mode and self.config.enable_cuda_graph and inference_context.is_static_batching():
+            raise ValueError(f"CUDA graphs must use flash decode with static batching!")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)